src/lib/thanos/types.ts

```diff
@@ -21,6 +24,21 @@ export enum ThanosAccountType {
   Connected
 }
 
+export interface ThanosNetwork {
+  id: string | number;
```

Comment:
Why `string | number`? Shouldn't it be just `string`

Reply:
Not necessary :)
This ThanosNetwork defined locally, any term that can be used for `===` equality is valid.
But I'll change this to just "string". It's more clear for reading because strict.